### PR TITLE
Make the PDOK type configurable

### PR DESCRIPTION
**Pull request by boris-arkenaar**
closes Signalen/product-steering#107

The `type` for the call to PDOK was hardcoded to `adres`. Some municipalities need to be able to have roads as suggestions as well. (There exist roads without any specific address on them.) This PR makes that value configurable.

### CODE REVIEW

File: src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx [hunk 1]
```diff
@@ -67,6 +68,34 @@ describe('components/PDOKAutoSuggest', () => {
...
+      await renderAndSearch('Dam', {
+        fieldType: '',
+      })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.not.stringContaining(fieldTypeQs),
+        expect.objectContaining({ method: 'GET' })
+      )
+    })
+  })
+
```
Comment (kiruys) [inline src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx:98]: We should also test the response. So in the first test case, addresses are returned, and in the second case addresses and roads.

Comment (kiruys) [inline src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx:98]: We should also test the response. So in the first test case, addresses are returned, and in the second case addresses and roads.

File: src/components/PDOKAutoSuggest/PDOKAutoSuggest.tsx [hunk 2]
```diff
@@ -40,16 +41,26 @@ export interface PDOKAutoSuggestProps
 const PDOKAutoSuggest: FC<PDOKAutoSuggestProps> = ({
   fieldList = [],
   municipality = configuration.map.municipality,
+  fieldType = configuration.map.fieldType,
   ...rest
 }) => {
-  const fq = municipality
+  const fieldTypeParam = fieldType
+    ? [['fq', `${fieldTypeFilterName}:(${fieldType})`]]
+    : []
```
Comment (kiruys) [inline src/components/PDOKAutoSuggest/PDOKAutoSuggest.tsx:49]: Perhaps an idea to have 'adres' as default, and use fieldType when it is something else than 'adres'

Comment (kiruys) [inline src/components/PDOKAutoSuggest/PDOKAutoSuggest.tsx:49]: Perhaps an idea to have 'adres' as default, and use fieldType when it is something else than 'adres'

Comment (boris-arkenaar) [reply src/components/PDOKAutoSuggest/PDOKAutoSuggest.tsx:49]: The default value of `adres` is set here https://github.com/Amsterdam/signals-frontend/blob/9ba5c6654942fdbf86d1e90041f02be423488ad6/app.base.json#L59

File: src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx (hunk 1 above)
Comment (boris-arkenaar) [reply src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx:98]: The response is the same in both cases.

**Comment by boris-arkenaar** [comment]
This will need changes to the backend as well.

**Pull request closed without merging**